aliyun_img_utils/aliyun_cli.py

```diff
@@ -219,16 +220,53 @@ def upload(
 
 
 @click.command()
+@click.option(
+    '--image-name',
+    type=click.STRING,
+    required=True,
+    help='Name of the newly created compute image.'
+)
+@click.option(
+    '--image-description',
+    type=click.STRING,
+    required=True,
+    help='Description for the newly created image.'
+)
+@click.option(
+    '--platform',
+    type=click.STRING,
+    required=True,
+    help='The distribution of the image operating system.'
+)
 @click.option(
     '--blob-name',
     type=click.STRING,
-    help='Name of the blob in the storage bucket. '
-         'To be deleted.',
-    required=True
+    required=True,
+    help='Name to use for blob in the storage bucket. By default '
+         'the filename from image file will be used.'
```

Comment:
I think it would be nice if the default would be to take the image name. That creates a "by name" correlation of the blog in the storage with the image without having to keep track of things. For cases where only upload functionality is used defaulting to the source file name is what I would expect.

Reply:
Ack I added that to the help message but did not implement it as so.

Reply:
NM, this is only create. It does not include upload so we need both values. The blob name is set on upload and is more likely to be based on the image file name as opposed to the image name. So I will fix the help message instead.

Reply:
Example upload `/path/to/image/test-image-123.qcow2` would default create a blob as `test-image-123.qcow2` and image name would be like `test-image-123-v20210302`. So the default blob name handling is during upload and on create we need both blob name and image name.